src/chain.h

```diff
@@ -324,6 +324,11 @@ class CDiskBlockIndex : public CBlockIndex
             READWRITE(VARINT(nDataPos));
         if (nStatus & BLOCK_HAVE_UNDO)
             READWRITE(VARINT(nUndoPos));
+        uint8_t tag = 0;
```

Comment:
Will we be removing this post RC1?  We don't need this in production, right?  If so, please file a ticket for this.